### PR TITLE
feat: Strings Palindrome

### PR DESCRIPTION
**Pull request by v1ktoryy**
related to: #14

### CODE REVIEW

**Comment by ParthS007** [comment]
This pull request **introduces 4 alerts** when merging 816351cdfc2bd16d1ff3a58f365efc858cc7237f into 2d503575f6cb8169a0a88748c3771e29897bb039 - [view on LGTM.com](https://lgtm.com/projects/g/TheAlgorithms/Javascript/rev/pr-ebdc0ff3406b04dcd16cb66c57238d0c6e206d61)

**new alerts:**

* 4 for Missing variable declaration

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ParthS007** [comment]
This pull request **introduces 4 alerts** when merging 78ab5645ec9b03a1668d9c8ab9eaf825600f8851 into 2d503575f6cb8169a0a88748c3771e29897bb039 - [view on LGTM.com](https://lgtm.com/projects/g/TheAlgorithms/Javascript/rev/pr-dbfa4bb3f4be72a448ab4eb994c03c3298229760)

**new alerts:**

* 4 for Missing variable declaration

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ParthS007** [comment]
This pull request **introduces 1 alert** when merging 062ca1f6fce55962132d876824afa67ae922b206 into 2d503575f6cb8169a0a88748c3771e29897bb039 - [view on LGTM.com](https://lgtm.com/projects/g/TheAlgorithms/Javascript/rev/pr-58ba7fde01f7db950663a8f74458ab78793ef1e6)

**new alerts:**

* 1 for Assignment to constant

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: Strings/palindrome.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+var isPalindrome = function (text){
```
Comment (ms10398) [inline Strings/palindrome.js]: Please try to use ES6 standards

**Comment by ms10398** [comment]
Reopen it when the changes are done

**Pull request closed without merging**